src/server/page_server.cpp

```diff
@@ -205,6 +206,13 @@ page_server::connection_handler::receive_start_catch_up (tran_server_conn_t::seq
 
   er_log_debug (ARG_FILE_LINE, "receive_start_catch_up: hostname = %s, port = %d, LSA = (%lld|%d)\n", host.c_str (), port,
 		LSA_AS_ARGS (&catchup_lsa));
+  if (port == -1)
+    {
+      return; // TODO: It means that the ATS is booting up, it will be set properly after ATS recovery is implemented.
+    }
+
+  // TODO: A thread will take the catch-up including establishing connection to avoid blocking ATS->PS reqeusts.
+  m_ps.connect_to_leader_page_server (std::move (host), port);
```

Comment:
I think error handling is required when `connect_to_leader_page_server ()` is failed.
it might need to respond to ATS with an error code. 

Would this be addressed in other issue ?

Reply:
yes, in [LETS-635](http://jira.cubrid.org/browse/LETS-635)
Stabilize Multi-PS and mitigate its restrictions.
```
Handle the exceptions during the catch-up process. ([LETS-632](http://jira.cubrid.org/browse/LETS-632))
The target PS may truncate some of log pages requested.
The target PS may be disconnected normally or abnormally.
And any error during the process like corrupting log pages for some reason.
```

Errors or disconnection from the leader PS are detected in another issue in thie epic, but it's going to be addressed in the above epic. I think, in any case, the follower PS re-request the catch-up information to the ATS.

Reply:
OK! 
`The target PS may be disconnected normally or abnormally.` 
Then, this should include scenarios where the leader PS is shut-down before catch-up initiation or if the connection with the leader PS is disconnected during catch-up

Reply:
Right, it's described in http://jira.cubrid.org/browse/LETS-784, which will be handled in this epic.